bbc/simorgh
# Service specific manifests

Resolves #1134 #1135

**Overall change:** Allows service-specific manifest files to be served by the application.
e.g. at http://localhost:7080/news/articles/manifest.json & http://localhost:7080/persian/articles/manifest.json

**Code changes:**

- Moved News Manifest file & associated icons into `public/news/`
- Added Persian manifest file & icons to `public/persian`
- Moved Manifest `<link>` tag to Articles container, to be able to use the `service` value. 
- Added new route to handle a request to `/:service/articles/manifest.json` to serve file from location `public/:service/manifest.json`


**Test instructions **
`npm run build && npm run start` to ensure you build the static files from `public` into `build/public` and then serve these. 

---
